### PR TITLE
Apply some more netCDF metadata tweaks

### PR DESCRIPTION
**Pull request by jhkennedy**
Relevant discussion in ITS_LIVE slack: https://itslive.slack.com/archives/C01BF2BCVJ8/p1637339972267500

### CODE REVIEW

File: hyp3_autorift/vend/README.md [hunk 1]
```diff
@@ -35,3 +35,9 @@ We've replaced it  with `hyp3_autorift.io.get_topsinsar_config`.
      for Sentinel-1 products
    * provide `sensor_img*=MSI` in the NETCDF metadata for Sentinel-2 products;
      these changes should be included in the next autoRIFT release.
+2. The changes listed in `CHANGES-METADATA-2.diff` were applied to
+   * change the `flag_stable_shit*` netCDF attributes to `stable_shift_flag*` so
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/README.md:39]:
```suggestion
   * change the `flag_stable_shift*` netCDF attributes to `stable_shift_flag*` so
```

File: hyp3_autorift/vend/netcdf_output.py [hunk 2]
```diff
@@ -655,17 +655,17 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
         var.setncattr('stable_shift_slow',np.nan)
 
     if stable_count != 0:
-        var.setncattr('vx_error_mask',int(round(vx_error_mask*10))/10)
+        var.setncattr('verror_mask',int(round(vx_error_mask*10))/10)
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/netcdf_output.py:658]:
```suggestion
        var.setncattr('error_mask',int(round(vx_error_mask*10))/10)
```

File: hyp3_autorift/vend/README.md (hunk 1 above)
Comment (jhkennedy) [reply hyp3_autorift/vend/README.md:39]: :eyes:

File: hyp3_autorift/vend/netcdf_output.py [hunk 3]
```diff
@@ -947,17 +947,17 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
             var.setncattr('stable_shift_slow',np.nan)
 
         if stable_count != 0:
-            var.setncattr('va_error_mask',int(round(va_error_mask*10))/10)
+            var.setncattr('error_mask',int(round(va_error_mask*10))/10)
         else:
-            var.setncattr('va_error_mask',np.nan)
-        var.setncattr('va_error_mask_description','RMSE over stable surfaces, stationary or slow-flowing surfaces with velocity < 15 m/yr identified from an external mask')
+            var.setncattr('error_mask',np.nan)
+        var.setncattr('rror_mask_description','RMSE over stable surfaces, stationary or slow-flowing surfaces with velocity < 15 m/yr identified from an external mask')
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/netcdf_output.py:953]:
```suggestion
        var.setncattr('error_mask_description','RMSE over stable surfaces, stationary or slow-flowing surfaces with velocity < 15 m/yr identified from an external mask')
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**